### ISSUE TITLE
[Feature Request] Simple File Manager in App

### ISSUE THREAD

**Issue by GMak9** [tachiyomiorg/tachiyomi#4271]
Some considerations regarding the issue #1767

Even if you clear the database, deleting data from sleeves that are not in the library will not cause chapters that were downloaded in half, of a manga that is still part of the library, to be deleted. Chapters to which download errors occur, incomplete download, take up space without being assigned the download badget, this type of occurrence will take up a lot of space in the future. 

The right way to manage space would be to integrate a file manager, which in other words checks in each Manga/comic show each byte of space occupied by it. 

Another solution, in the bottom tab or next to the download tab, whatever, a file manager will inform, in list, how much each Manga is taking up space, considering only the data occupied by the chapters, even if only a few bytes
Without further ado, this type of implementation has as reference the App "Manga Bird", it integrates the file manager, Amaze, however I suggest the implementation without being used external sources, leaving it light.

**Comment by Noriller** [comment]
If I could make an addendum:

I would like that this feature could also clear those manga that I've already read but didn't have any updates in X time (maybe 30~90 days)

I have a lot of manga that I've read every chapter and that have downloaded chapters but it's been ages since the last chapter.
This could also clean those random chapters that weren't deleted by the "delete after read" feature for any reason.

**Comment by arkon** [comment]
@Noriller The unread + downloaded badges in the library can serve that purpose. Since you can see if there's stray downloads (i.e. # of downloads ≠ # of unread chapters), you can use the delete chapters function (long press library items > delete icon > select downloaded chapters only) there to remove them.

**Comment by Noriller** [comment]
@arkon if I had just a couple dozen, sure. but when hitting hundreds... it can be a bit tiresome to do that...

(last time I've needed space I just deleted the whole downloads folder and downloaded what I wanted as I needed)

**Comment by arkon** [comment]
It's the best in-app workaround at the moment, so I thought it'd be worth mentioning.

A file manager is realistically not going to happen anytime soon as it'd complicate the app a lot (_especially_ in Android 11+), and integrating a third-party one (e.g. Amaze, as the OP mentioned) is slightly pointless considering you could just use the file manager app separately.

**Comment by GMak9** [comment]
I don't expect something big, like a file manager, but rather a way to identify remnants of pages occupying storage and a way to clean it up. 
A more complete and multifunctional download manager. The example I used was the Manga Bird app, another application that brings a simpler and closer use to what I imagine is the MangaZone App download page. It turns out that in both cases it has a similar UX and equally pleasantThe example I used was the Manga Bird app, another application that brings a simpler and closer use to what I imagine is the MangaZone App download page. It turns out that in both cases it has a similar ux and equally nice.
